src/background/index.ts

```diff
@@ -242,6 +288,13 @@ const messageListener = (request) => {
       // Send a message to the active tab
       agent.Primers.create({ title: request.title }).then(response => {
         chrome.tabs.sendMessage(activeTab.id, {message: "create_primer_response", body: response });
+
+        window.analytics.track('Collection Created', {
+          title: response.primer.title,
+          userId: response.primer.users[0].id,
+          context: 'extension'
```

Comment:
let's change this to 'midas' to keep things consistent
```suggestion
          context: 'midas'
```